### ISSUE TITLE
Unable to hide repository issues

### ISSUE THREAD

**Issue by geoah** [ptsochantaris/trailer#443]
Hey there,

I'm on 1.7.6 (1584) and I'm seeing a weird issue trying to hide repository issues.

It seems tha the "Hiden" option in the drop down doesn't exist on individual repositories, and if I try to hide issues for all or filtered repositories, it correctly filters them until the next refresh, (or until I close the settings window which seems to trigger a refresh.)

![2021-08-09 23 30 56](https://user-images.githubusercontent.com/88447/128782822-1b916bdd-e2b5-4a29-a8b8-2dfaa5e50a16.gif)

I played around with settings a bit but couldn't find something that might be causing this.
If there is anything more I can give in terms of feedback to help with this let me know.

Thank you for all your hard work on Trailer :)

**Comment by ptsochantaris** [comment]
Hi @geoah and thanks for reaching out.

What you're seeing isn't technically a bug, more like slightly <cough> unoptimal UI on my end. "Authored" is basically the same as "Hide". The reason the indicator there changes to "Authored" is that Trailer has detected you have items authored in those repositories, and that you have selected the option "sync PRs/issues I have created in any repository" from the advanced panel in the repos tab.

So basically:
`Hidden` means Trailer won't bother with that repository for that type of items
`Authored` means that Trailer won't query items from that repository explicitly, but there are some that will appear on your list as you're the author of them.

Or at least that's what supposed to happen :D If that's not the behaviour you're seeing let me know and we can dig further :)

I hope this helps!

**Comment by geoah** [comment]
Hey @ptsochantaris,

Yap removing "sync PRs/issues I have created in any repository" indeed was the culprit.
Sorry for the non-bug report, it does make sense.

I'll close this ticket now.

Thank you for all your work on trailer! :D

**Comment by mcg** [comment]
Went round and round with this and glad I found this issue. Maybe that feature needs a note or something to explain what it will do with the repo filtering.
